### ISSUE TITLE
Text Editor slow with this extension enabled in large solutions

### ISSUE THREAD

**Issue by Yelmond** [mike-ward/VSColorOutput#15]
The Text Editor becomes border-line unusable on a large solution if I enable this extension. Every change to a text file causes VS to use 100% of a core for about 0.5 seconds. Deleting characters gets close to 1sec delay for every character.

This is on Visual Studio 2015 Pro, Windows 10, with a large C++ heavy solution (ex. all of boost), using latest version of VSColorOutput 2.1.1.

Great extension otherwise so I'm sad I have to disable it.

**Comment by mike-ward** [comment]
It's not that I don't believe you, it's just that the extension has nothing to do with the text editor. Can't even begin to think why this would be the case.

Still I downloaded boost and tried a couple of projects. I didn't see any slow downs. Was there a particular solution/project you were using?

**Comment by Yelmond** [comment]
My apologies, I spoke too soon. Although the plugin does aggravate the issue, I'm fairly sure at this point that it only does so because of the Visual Studio features it uses. I ran into the same problem later in the loading process with the VSColorOutput extensions uninstalled.

The problem does not seem to happen in SafeMode but I'm still trying to figure out which extension is the bugger.

Anyway, sorry for the false alarm. Closing issue.
